src/config.ts

```diff
@@ -15,7 +16,7 @@ function onErrorRetry(
   revalidate: Revalidator,
   opts: Required<RevalidatorOptions>
 ): void {
-  if (!webPreset.isDocumentVisible()) {
+  if (!webPreset.isVisible()) {
```

Comment:
it's not a documented option though, we can mention it in this PR as part of release notes. maybe some users are using custom `isDocumentVisible` now. like RN users

Reply:
Yeah, I think I'll just revert this change for now until we have an ultimate solution for RN.